easybuild/easyconfigs/l/LncLOOM/LncLOOM-2.0-environment.patch

```diff
@@ -0,0 +1,51 @@
+Binary files LncLOOMv2/.git/index and LncLOOMv2-edit/.git/index differ
+diff -ru LncLOOMv2/LncLOOMv2/LncLOOM.py LncLOOMv2-edit/LncLOOMv2/LncLOOM.py
+--- LncLOOMv2/LncLOOMv2/LncLOOM.py	2021-02-25 14:54:12.572901247 +0100
++++ LncLOOMv2-edit/LncLOOMv2/LncLOOM.py	2021-02-25 15:05:16.159414207 +0100
+@@ -115,14 +115,14 @@
+ 
+ 
+     if args.eclip== True:
+-        efile = os.path.join(os.path.dirname(__file__), 'src', 'for_eclip_annotation.txt')
++        efile = os.environ['LNCLOOM_ECLIP_CONFIG']
```

Comment:
This will crash hard if `$LNCLOOM_ECLIP_CONFIG` is not defined, with an ugly traceback.

Maybe use the value if it's defined, but fall back to what it was before by default?

```python
default_efile = os.path.join(os.path.dirname(__file__), 'src', 'for_eclip_annotation.txt')
efile = os.environ.get('LNCLOOM_ECLIP_CONFIG', default_efile)
```

Likewise below, of course.

Reply:
The thing is, the default value doesn't work at all in this case, the `for_eclip_annotation.txt` assumes the supporting data is inside site-packages, which cannot be the case here. I'll add a better error message instead, I think

Reply:
OK, you should definitely avoid a hard ugly crash if `$LNCLOOM_ECLIP_CONFIG` is not defined.